File: multipicker/src/main/java/com/edwardstock/multipicker/internal/views/ErrorView.java
package com.edwardstock.multipicker.internal.views;

import com.edwardstock.multipicker.internal.mvp.MvpView;

/**
 * android-multipicker. 2018
 * @author Eduard Maximovich [edward.vstock@gmail.com]
 */
public interface ErrorView extends MvpView {
    void onError(CharSequence error);
    void onError(Throwable t);
}

File: multipicker/src/main/java/com/edwardstock/multipicker/internal/views/ErrorView.kt
package com.edwardstock.multipicker.internal.views

/**
 * android-multipicker. 2018
 * @author Eduard Maximovich [edward.vstock@gmail.com]
 */
interface ErrorView {
    fun onError(error: CharSequence?)
    fun onError(t: Throwable?)
}